components/support/rc_crypto/nss/nss_sys/src/bindings/prtypes.rs

```diff
@@ -2,11 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use std::os::raw::{c_int, c_uint, c_ulong};
+use std::os::raw::{c_int, c_uint};
 
 pub type PRIntn = c_int;
 pub type PRBool = PRIntn;
-pub type PRUword = c_ulong;
+pub type PRUword = usize;
```

Comment:
https://searchfox.org/mozilla-central/source/nsprpub/pr/include/prtypes.h#470 indicates that prword/pruword is the same.